tests/test_xyzzyx.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+#import ndio.remote.neurodata as neurodata
+from ndio.remote.data import data
```

Comment:
So for the refactoring the idea is that everything should still be backwards compatible, in other words to the user there should be no change in interface, there is just a change in the underlying code. This means that a lot of method calls will just become something like:

def getCutout(kwargs):
  self.data.getCutout(**kwargs)

So for these things in test we should still just be importing and using remote.neurodata